src/gui/BandSplitter/BandSplitterPlot.cpp

```diff
@@ -208,5 +242,44 @@ void BandSplitterPlot::resized()
     const auto bounds = getLocalBounds();
     cutoffSlider.setBounds (bounds);
     cutoff2Slider.setBounds (bounds);
+    int x1 = cutoffSlider.getThumbBounds().getX();
+    int x2 = cutoff2Slider.getThumbBounds().getX();
+    int spectrumWidth = x2 - x1;
+    if (spectrumWidth > 0)
+        spectrumAnalyser.setBounds (x1, 0, spectrumWidth, bounds.getHeight());
+    else
+        spectrumAnalyser.setBounds (0, 0, 0, 0);
```

Comment:
Hmm, I wonder if there's anything wrong with letting the analyzer cover the full bounds here? There will be a little bit of "bleed" between the bands (especially for the lower-order filters), so it would be useful to show that.